prediction_market_agent/agents/metaculus_agent/deploy.py

```
@@ -1,84 +1,83 @@
+        if not self.repeat_predictions and market.have_predicted:
+            return False
 
-            if answer is None:
-                logger.error("No answer was given. Skipping")
-            else:
-                market.submit_prediction(
-                    p_yes=answer.p_yes,
-                    reasoning=check_not_none(answer.reasoning),
-                )
+        # Otherwise all markets on Metaculus are fine.
+        return True
```

Ruff

Return the negated condition directly

(SIM103)